engine/src/main/resources/assets/i18n/menu_cs.lang

```diff
@@ -1,316 +1,459 @@
-    "warning": "Varování",
-    "water-reflections": "Odrazy na vodě",
-    "water-reflections-global": "Vše",
-    "water-reflections-sky": "Jen nebe",
-    "water-reflections-ssr": "SSR (experimentální)",
-    "widget-selection-prompt": "Vyber typ widgetu:",
-    "widget-selection-title": "Výběr Widgetu",
-    "world-config-preview": "Detaily...",
-    "world-seed": "Zdroj"
+  "accept": "Přijmout",
+  "activate-module": "Aktivovat",
+  "add": "Přidat",
+  "add-server": "Přidat nový",
+  "add-server-tip": "Stisknutím ENTER zkopíruješ jméno do adresy",
+  "advance": "Advance",
+  "advanced": "Pokročilé",
+  "advanced-game-setup": "Pokročilé nastavení hry",
+  "advanced-usage": "Pokročilé použití!",
+  "animate-grass": "Animovaná tráva",
+  "animate-water": "Animovaná voda",
+  "audio-settings": "Zvuk",
+  "audio-settings-title": "Nastavení zvuku",
+  "back": "Zpět",
+  "behavior-editor-assign": "Přiradit",
+  "behavior-editor-copy": "Kopírovat",
+  "behavior-editor-layout": "Zarovnat",
+  "behavior-editor-new": "Nové",
+  "behavior-editor-paste": "Vložit",
+  "behavior-editor-remove": "Odebrat",
+  "binding-attack": "Útok",
+  "binding-autoMove-mode": "Přepnout automatickou chůzi",
+  "binding-backwards": "Dozadu",
+  "binding-behavior-editor": "Přepnout editor chování",
+  "binding-chat": "Přepnout chat",
+  "binding-console": "Přepnout konzoli",
+  "binding-crouch": "Přikrčit / Dolů",
+  "binding-crouch-mode": "Režim přikrčení",
+  "binding-decrease-view-distance": "Snížit viditelnou vzdálenost",
+  "binding-forwards": "Dopředu",
+  "binding-frob": "Použít cílový předmět",
+  "binding-hide-hud": "Skrýt HUD",
+  "binding-increase-view-distance": "Zvýšit viditelnou vzdálenost",
+  "binding-jump": "Skok / Nahoru",
+  "binding-left": "Úkrok vlevo",
+  "binding-nui-editor": "NUI Editor",
+  "binding-nui-skin-editor": "NUI Editor vzhledu",
+  "binding-pause": "Herní menu",
+  "binding-right": "Úkrok vpravo",
+  "binding-screenshot": "Pořídit screenshot",
+  "binding-show-online-players": "Zobrazit online hráče",
+  "binding-toggle-speed-permanently": "Trvale přepnout rychlost",
+  "binding-toggle-speed-temporarily": "Dočasně přepnout rychlost",
+  "binding-toolbar-1": "Toolbar Slot 1",
+  "binding-toolbar-2": "Toolbar Slot 2",
+  "binding-toolbar-3": "Toolbar Slot 3",
+  "binding-toolbar-4": "Toolbar Slot 4",
+  "binding-toolbar-5": "Toolbar Slot 5",
+  "binding-toolbar-6": "Toolbar Slot 6",
+  "binding-toolbar-7": "Toolbar Slot 7",
+  "binding-toolbar-8": "Toolbar Slot 8",
+  "binding-toolbar-9": "Toolbar Slot 9",
+  "binding-toolbar-10": "Toolbar Slot 10",
+  "binding-use-item": "Použít držený předmět",
+  "biome-fog": "Mlha",
+  "biome-temperature": "Teplota",
+  "biome-humidity": "Vlhkost",
+  "biome-name": "Název",
+  "bloom-effect": "Rozostření",
+  "bobbing": "Pohyb hlavy",
+  "button-web-browser-confirmation-title": "Je vyžadováno potvrzení.",
+  "button-web-browser-confirmation-message": "Opravdu chceš otevřít toto url v prohlížeči?",
```

Comment:
```suggestion
  "button-web-browser-confirmation-message": "Opravdu chceš otevřít tuto adresu v prohlížeči?",
```

Reply:
Sounds way better, maybe we should use the `address` also in the EN translation which I've used as reference as well?